client-registry-jempi/swarm.sh

```diff
@@ -86,6 +86,15 @@ function initialize_package() {
       log warn "Service 'interoperability-layer-openhim' does not appear to be running... skipping configuring of async/sync JeMPI channels"
     fi
 
+    if [[ "${ACTION}" == "init" ]]; then
+      log info "initiate bootstrapper data resetAll"
+      if docker service ps -q jempi_jempi-bootstrapper &>/dev/null; then
+        try "docker exec -i $(docker ps -q -f name=jempi_jempi-bootstrapper) ./bootstrapper.sh data resetAll" throw "Could not initiate bootstrapper "
```

Comment:
One thing I notice is that this outputs some text into the console when starting up platform. It would be best to suppress this but not a big deal.

Reply:
I am aware of this. Its the logs. I need to find a solution to supress this